.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "src/test/resources/network/brightspots/rcv/test_data"]
-	path = src/test/resources/network/brightspots/rcv/test_data
-	url = https://github.com/brightspots/rcvtests.git
```

Comment:
There's also mention of "submodules" in the contributor guide [here](https://github.com/BrightSpots/rcv/blob/88afa608a615b4229e6e758b0672f673b3b98697/contributor_guide.md#L22) and in the `test.yaml` file [here](https://github.com/BrightSpots/rcv/blob/26ae62fdd9402d9fd0d3f435771fb90553843dd1/.github/workflows/test.yml#L12) that might need to be addressed.

Reply:
I remove the reference from the guide.  

The yaml I imagine is just a generic configuration to configure submodules recursively if there are any.  It doesn't seem to have impacted the test from running and completing which is the important thing here.  @artoonie any thoughts on this?